groovy/postInit/chemistry/inorganic_chemistry/elements/p_block/group13/BoronChain.groovy

```diff
@@ -139,8 +139,7 @@ FLUIDIZED_BR.recipeBuilder()
 
 FLUIDIZED_BR.recipeBuilder()
     .inputs(ore('dustElectrolyticBoron') * 1)
-    .fluidInputs(fluid('chlorine') * 3040)
-    .chancedOutput(metaitem('dustMagnesiumChloride') * 3, 600, 0)
+    .fluidInputs(fluid('chlorine') * 3000)
```

Comment:
do not delete MgCl2 Output - this recipe is purifying Electrolytic Boron from Mg impurities

Reply:
Mg existed even if electrolytic B doesn't use Mg in its processing